dnsapi/dns_bluemix.sh

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env sh
+#
+# DNS Integration for IBM Bluemix (formerly SoftLayer)
+#
+# Author: luizgn
+# Based on sample from Neilpang
+# Report Bugs here: https://github.com/luizgn/acme.sh
+#
+########  Public functions #####################
+
+domainId=
+domain=
+host=
+recordId=
+
+dns_bluemix_add() {
+  fulldomain=$1
+  txtvalue=$2
+
+  _info "Attempting to add ${fulldomain} with ${txtvalue} into Bluemix's DNS."
+
+  # Curl is required
+  if ! type curl >/dev/null; then
+    _err "curl missing. Please isntall curl"
+    return 1
+  fi
+
+  # BLUEMIX_USER is required
+  if [[ "${BLUEMIX_USER}" == "" ]]; then
+    _err "Environment variable BLUEMIX_USER not defined"
+    return 1
+  fi
+
+  # BLUEMIX_KEY is required
+  if [[ "${BLUEMIX_KEY}" == "" ]]; then
+    _err "Environment variable BLUEMIX_KEY not defined"
+    return 1
+  fi
+
+  # Get right domain and domain id
+  getDomain ${fulldomain}
+
+  # Did we find domain?
+  if [[ "${domain}" == "" ]]; then
+    return 1
```

Comment:
use `_err()` function to output a error message here also, instead of just returning 1.